src/components/media-loader.js

```diff
@@ -182,9 +182,7 @@ AFRAME.registerComponent("media-loader", {
       } else {
         el.setAttribute("ammo-shape", {
           type: SHAPE.BOX,
-          halfExtents: { x: 0.5, y: 0.5, z: 0.02 },
```

Comment:
So what is now responsible for setting the right half extents?

Reply:
three-to-ammo does it automatically